src/transformers/adapters/model_mixin.py

```diff
@@ -339,28 +345,34 @@ def save_adapter(
     def save_adapter_fusion(
         self,
         save_directory: str,
-        adapter_names: list,
+        adapter_names: Union[Fuse, list, str],
         meta_dict: dict = None,
         custom_weights_loaders: Optional[List[WeightsLoader]] = None,
     ):
         """
-        Saves an adapter and its configuration file to a directory so that it can be shared or reloaded using
-        `load_adapter()`.
+        Saves an AdapterFusion layer and its configuration file to a directory so that it can be shared or reloaded
+        using `load_adapter_fusion()`.
 
         Args:
-            save_directory (str): Path to a directory where the adapter should be saved.
-            adapter_name (str): Name of the adapter to be saved.
+            save_directory (str): Path to a directory where the AdapterFusion should be saved.
+            adapter_names (Union[Fuse, list, str]): AdapterFusion to be saved.
 
         Raises:
-            ValueError: If the given adapter name is invalid.
+            ValueError: If the given AdapterFusion name is invalid.
         """
+        if isinstance(adapter_names, Fuse):
+            adapter_fusion_name = ",".join(adapter_names.children)
+        elif isinstance(adapter_names, list):
+            adapter_fusion_name = ",".join(adapter_names)
+        else:
```

Comment:
Maybe it would make sense to check whether this is really a string in the else case and throw an error otherwise